src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -65,11 +72,21 @@ public static JsonArray createDeviceStatusDualBand(int channel2G, int txPower2G,
 		return jsonList;
 	}
 
+	private static int channelToFrequency(int channel) {
```

Comment:
add some error checking here or at appropriate public call-sites for invalid channels -- perhaps, add list of valid channels to docstring